src/test/java/com/google/api/codegen/testdata/csharp/csharp_library.baseline

```diff
@@ -10251,62 +10251,66 @@ namespace Google.Example.Library.V1
-            Modify_ApiCall_AddComments(ref _callAddComments);
+            Modify_AddCommentsApiCall(ref _callAddComments);
             Modify_ApiCall(ref _callGetBookFromArchive);
-            Modify_ApiCall_GetBookFromArchive(ref _callGetBookFromArchive);
+            Modify_GetBookFromArchiveApiCall(ref _callGetBookFromArchive);
             Modify_ApiCall(ref _callGetBookFromAnywhere);
-            Modify_ApiCall_GetBookFromAnywhere(ref _callGetBookFromAnywhere);
+            Modify_GetBookFromAnywhereApiCall(ref _callGetBookFromAnywhere);
             Modify_ApiCall(ref _callGetBookFromAbsolutelyAnywhere);
-            Modify_ApiCall_GetBookFromAbsolutelyAnywhere(ref _callGetBookFromAbsolutelyAnywhere);
+            Modify_GetBookFromAbsolutelyAnywhereApiCall(ref _callGetBookFromAbsolutelyAnywhere);
             Modify_ApiCall(ref _callUpdateBookIndex);
-            Modify_ApiCall_UpdateBookIndex(ref _callUpdateBookIndex);
+            Modify_UpdateBookIndexApiCall(ref _callUpdateBookIndex);
             Modify_ApiCall(ref _callStreamShelves);
-            Modify_ApiCall_StreamShelves(ref _callStreamShelves);
+            Modify_StreamShelvesApiCall(ref _callStreamShelves);
             Modify_ApiCall(ref _callStreamBooks);
-            Modify_ApiCall_StreamBooks(ref _callStreamBooks);
+            Modify_StreamBooksApiCall(ref _callStreamBooks);
             Modify_ApiCall(ref _callDiscussBook);
-            Modify_ApiCall_DiscussBook(ref _callDiscussBook);
+            Modify_DiscussBookApiCall(ref _callDiscussBook);
             Modify_ApiCall(ref _callFindRelatedBooks);
-            Modify_ApiCall_FindRelatedBooks(ref _callFindRelatedBooks);
+            Modify_FindRelatedBooksApiCall(ref _callFindRelatedBooks);
             Modify_ApiCall(ref _callAddTag);
-            Modify_ApiCall_AddTag(ref _callAddTag);
+            Modify_AddTagApiCall(ref _callAddTag);
             Modify_ApiCall(ref _callAddLabel);
-            Modify_ApiCall_AddLabel(ref _callAddLabel);
+            Modify_AddLabelApiCall(ref _callAddLabel);
             Modify_ApiCall(ref _callGetBigBook);
-            Modify_ApiCall_GetBigBook(ref _callGetBigBook);
+            Modify_GetBigBookApiCall(ref _callGetBigBook);
             Modify_ApiCall(ref _callGetBigNothing);
-            Modify_ApiCall_GetBigNothing(ref _callGetBigNothing);
+            Modify_GetBigNothingApiCall(ref _callGetBigNothing);
             Modify_ApiCall(ref _callTestOptionalRequiredFlatteningParams);
-            Modify_ApiCall_TestOptionalRequiredFlatteningParams(ref _callTestOptionalRequiredFlatteningParams);
+            Modify_TestOptionalRequiredFlatteningParamsApiCall(ref _callTestOptionalRequiredFlatteningParams);
             OnConstruction(grpcClient, effectiveSettings, clientHelper);
         }
 
+        // Partial methods are named to (mostly) ensure there cannot be conflicts with RPC method names.
+
+        // Partial methods called for every ApiCall on construction.
```

Comment:
Nit: *maybe* we should specify that the generic call is made before the specific one?
(I'm not overly keen on "modification of all the underlying ApiCall objects" given that they're immutable, but it makes sense with the Modify method name, and I suspect it'll be clear enough to everyone actually using this facility.)